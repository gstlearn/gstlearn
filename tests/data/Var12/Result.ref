
Variogram characteristics
=========================
Number of variable(s)       = 1
Number of direction(s)      = 2
Space dimension             = 3
Variable(s)                 = [Unknown]
Variance-Covariance Matrix     0.972

Direction #1
------------
Number of lags              = 10
Direction coefficients      =      1.000     0.000     0.000
Direction angles (degrees)  =      0.000     0.000     0.000
Tolerance on direction      =      0.500 (degrees)
Calculation lag             =     15.000
Tolerance on distance       =     50.000 (Percent of the lag value)

For variable 1
      Rank    Npairs  Distance     Value
         03754860.000     4.761     0.251
         131371100.000    15.837     0.476
         255914100.000    30.221     0.658
         373723600.000    44.952     0.768
         486886900.000    59.928     0.833
         590329400.000    74.888     0.858
         691536900.000    89.795     0.871
         787299500.000   104.772     0.922
         879866700.000   119.751     0.999
         969764900.000   134.672     1.025

Direction #2
------------
Number of lags              = 4
Direction coefficients      =      0.000     0.000    -1.000
Direction angles (degrees)  =      0.000     0.000    90.000
Tolerance on direction      =      0.500 (degrees)
Calculation lag             =      1.000
Tolerance on distance       =     50.000 (Percent of the lag value)

For variable 1
      Rank    Npairs  Distance     Value
         1199148.000     0.742     0.297
         2172636.000     1.740     0.571
         3146124.000     2.739     0.755

Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 1
Number of basic structure(s) = 2
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Exponential
- Sill         =      0.694
- Ranges       =     46.405    46.405     5.220
- Theo. Ranges =     15.490    15.490     1.742
Spherical
- Sill         =      6.345
- Ranges       =   4084.874  4084.874    125.736
Total Sill     =      7.039
Known Mean(s)     0.000
Goodness-of-fit (as a percentage of the variance) corresponds to level #1 (1 for very good)
