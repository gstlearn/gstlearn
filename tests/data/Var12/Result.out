
Convergence not reached after 1000 iterations (6 parameters)

Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 1
Number of basic structure(s) = 4
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Nugget Effect
- Sill         =      0.098
Cubic
- Sill         =      0.186
- Ranges       =     13.953    13.953     4.024
Exponential
- Sill         =      0.214
- Ranges       =    184.467   184.467   138.649
- Theo. Ranges =     61.577    61.577    46.282
Spherical
- Sill         =      0.461
- Ranges       =     71.781    71.781     3.135
Total Sill     =      0.960
